### PR TITLE
Issue7 appendices

### PR DESCRIPTION
**Pull request by niellaNina**
This is just adding the appendices as they where in V1. As they are now, they are mostly MET specific and need to be generalized, but I think that is easier to do when we are further with the rest of the chapters.

### CODE REVIEW

File: doc/appendixA.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+|METSIS-ts
+|WPS or HTTP interface to graphical diagrams.
+|Not yet openly available, but beta-version is in use in ADC, SIOS, GCW, NorDataNet, YOPP and APPLICATE portals.
+
+|MMD XSD
+|XML Schema document for MMD
+|https://github.com/metno/mmd[github repository: mmd]
+
+|nc_to_mmd.pl
```
Comment (ElodieFZ) [inline doc/appendixA.adoc:57]:
```suggestion
|nc_to_mmd.py
```

File: doc/appendixB.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+[%header, cols=3*]
+|===
+|User category
+|Description
+|METNorway Examples
+
+|Public service collaborators
+|Use the data as part of collaborative public services. Includes public sector agencies and companies, government-supported enterprises.  Specific requirements on the data and delivery, possibly by SLA or OLA. 
+|Examples: NVE (flood warning service), NRK (yr.no, radio, TV), Norwegian Defence Research Establishment (FFI), Avinor (aviation forecasting), research infrastructures (NORMAP, SIOS, NorDataNet), Norwegian Public Roads Administration (SVV), The Norwegian Coastal Administration (BarentsWatch), JRCC (search-and-rescue support), Bane NOR (Railway Infrastructure Company), Norwegian Environment Agency, NERSC (Copernicus Marine Service), NIVA (), NILU (air quality monitoring), NRPA (radiation emergency response support), IMR (fish eggs/larvae dispersion)
```
Comment (ElodieFZ) [inline doc/appendixB.adoc:19]:
```suggestion
|Examples: NVE (flood warning service), NRK (yr.no, radio, TV), Norwegian Defence Research Establishment (FFI), Avinor (aviation forecasting), research infrastructures (NORMAP, SIOS, NorDataNet), Norwegian Public Roads Administration (SVV), The Norwegian Coastal Administration (BarentsWatch), JRCC (search-and-rescue support), Bane NOR (Railway Infrastructure Company), Norwegian Environment Agency, NERSC (Copernicus Marine Service), NIVA (Norwegian Institute for Water Research), NILU (air quality monitoring), NRPA (radiation emergency response support), IMR (fish eggs/larvae dispersion)
```

File: doc/appendixB.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+|Value-added service providers
+|Use open data in proprietary services, often commercial. Value-added and repackaging services. Weak requirements on the data. 
+|Little or no collaboration with MET Norway.
+
+Examples: StormGeo (forecasting, consultancy, Navtor (ship routing services), offshore operators, news media, Skiforeningen (iMarka), web and mobile app developers, …
```
Comment (ElodieFZ) [inline doc/appendixB.adoc:25]:
```suggestion
Examples: StormGeo (forecasting, consultancy), Navtor (ship routing services), offshore operators, news media, Skiforeningen (iMarka), web and mobile app developers, … 
```

File: doc/appendixB.adoc [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+|Little or no collaboration with MET Norway.
+
+Examples: StormGeo (forecasting, consultancy, Navtor (ship routing services), offshore operators, news media, Skiforeningen (iMarka), web and mobile app developers, … 
+
+|Commercial services
+|Data services provided under commercial contract, SLA, OLA, or similar. 
+|Examples: Statkraft, Copernicus (CMEMS, C3S, CAMS), offshore operators (EXWW/VXWW), ...
+
+|Direct consumers
+|Use data in their own activities (e.g., decision-making, research). Both public and private enterprises. Requirements on data and delivery vary; open data use is preferred;
```
Comment (ElodieFZ) [inline doc/appendixB.adoc:32]:
```suggestion
|Use data in their own activities (e.g., decision-making, research). Both public and private enterprises. Requirements on data and delivery vary; open data use is preferred.
```

File: doc/appendixB.adoc [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+Examples: StormGeo (forecasting, consultancy, Navtor (ship routing services), offshore operators, news media, Skiforeningen (iMarka), web and mobile app developers, … 
+
+|Commercial services
+|Data services provided under commercial contract, SLA, OLA, or similar. 
+|Examples: Statkraft, Copernicus (CMEMS, C3S, CAMS), offshore operators (EXWW/VXWW), ...
+
+|Direct consumers
+|Use data in their own activities (e.g., decision-making, research). Both public and private enterprises. Requirements on data and delivery vary; open data use is preferred; 
+|Halo has tailored datasets for public users.
```
Comment (ElodieFZ) [inline doc/appendixB.adoc:33]: What is Halo? It's not in the glossary as far as I can see.

File: doc/appendixB.adoc [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+Examples: academic researchers, students, commercial researchers, Defence (FFI), NGOs, DnV
+
+|Other catalogues and portals
+|Harvest metadata for inclusion in e.g. thematic catalogues and portals, with access to the data. Both public and private enterprises. No explicit collaboration. Require structured metadata and data access.
+|Examples: earthengine.google.com, research infrastructures,  …
+
+|Internal users
+|The organisation's own activities, covering the full value chain shown in Figure 1.  Requires structured data and metadata. 
+|Involves data used for both state-mandated and externally financed activities. Includes a majority of the institute’s co-workers in VDiv, SUV, FoU, ObsKlim and IT.
```
Comment (ElodieFZ) [inline doc/appendixB.adoc:43]: I'm not sure about this one. Should SUV, FoU ... be listed in the glossary? Probably not as it's met to be read by MET people.

**Review by ElodieFZ — CHANGES_REQUESTED** [review]

File: doc/appendixB.adoc (hunk 5 above)
Comment (niellaNina) [reply doc/appendixB.adoc:33]: I will add information about Halo into the glossary.

File: doc/appendixB.adoc (hunk 6 above)
Comment (niellaNina) [reply doc/appendixB.adoc:43]: I guess this is one of the parts that will only be available (in the end) in the MET version yes. Leave it as it is for now

**Pull request merged**